tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -302,3 +331,98 @@ def test_plot_pareto_front_invalid_axis_order(
             include_dominated_trials=include_dominated_trials,
             axis_order=invalid_axis_order,
         )
+
+
```

Comment:
Let's follow the same filtering rule as test functions in this file: 
```suggestion

@pytest.mark.filterwarnings("ignore::optuna.exceptions.ExperimentalWarning")
```